saleor/graphql/discount/mutations/utils.py

```diff
@@ -0,0 +1,23 @@
+from collections import defaultdict
+
+import graphene
+
+from saleor.discount.utils import CatalogueInfo
+from saleor.graphql.discount.mutations.sale_base_discount_catalogue import (
+    NodeCatalogueInfo,
+)
```

Comment:
Please use relative imports. Also, I think we could entirely drop `NodeCatalogueInfo` from Saleor as it doesn't sound too useful and understandable; instead, we could just use `DefaultDict[str, Set[str]]` wherever it's needed.

Reply:
I agree. fixed